packages/shared/styles/components/SfHeader.scss

```
@@ -1,11 +1,13 @@
 @import "../variables";
 
 $header-height: 3.125rem !default;
+$max-header-height: 6.25rem !default;
```

```suggestion
$header-max-height: 6.25rem !default;
```
Just to stick to conventions